src/components/Table/TableCell/TableCell.module.scss

```diff
@@ -1,4 +1,6 @@
 .tableCell {
-  padding: 9px 16px;
+  padding: 0 16px;
   overflow: hidden;
+  display: flex;
```

Comment:
hmm, seems like changing to flex caused an issue in the loading state
![image](https://github.com/mondaycom/monday-ui-react-core/assets/6093192/1e0f6071-0829-492f-9046-b076b9f83edb)

in that case, let's make sure that the Skeleton components inside get the correct height

Reply:
I have missed it during the test. It will be working properly when the 9px padding is reinstated